client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
```

Comment:
I see a lot of people forgetting their gitignore, these are really important, especially when we get to deployment.

Reply:
(I know it's in the project, they just leave it as is and it looks like you're adding to it)